### ISSUE TITLE
Naive switch from Flask to AIOHTTP fails

### ISSUE THREAD

**Issue by hjacobs** [spec-first/connexion#578]
I'm trying my example project (https://github.com/hjacobs/connexion-example) to switch to AIOHTTP, but it fails.

1. I have to add `only_one_api=True` to the constructor in `app.py` (this probably deserves a documentation entry)
2. server starts, but requests fail:

```
INFO:connexion.aiohttp_app:Listening on 0.0.0.0:8080..
======== Running on http://0.0.0.0:8080 ========
(Press CTRL+C to quit)
INFO:root:Creating pet 1..
ERROR:aiohttp.server:Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/aiohttp/web_protocol.py", line 381, in start
    resp = await self._request_handler(request)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp/web_app.py", line 322, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.6/dist-packages/aiohttp/web_middlewares.py", line 88, in impl
    return await handler(request)
  File "/usr/local/lib/python3.6/dist-packages/connexion/apis/aiohttp_api.py", line 32, in oauth_problem_middleware
    response = yield from handler(request)
  File "/usr/local/lib/python3.6/dist-packages/connexion/decorators/coroutine_wrappers.py", line 23, in wrapper
    connexion_response = yield from connexion_response
  File "/usr/local/lib/python3.6/dist-packages/connexion/apis/aiohttp_api.py", line 208, in get_response
    response.status, extra={'data': response.body, 'url': url})
AttributeError: 'tuple' object has no attribute 'status'
ERROR:aiohttp.server:Error handling request

**Comment by dutradda** [comment]
@hjacobs 
we know about it and we had discussed about that:
https://github.com/zalando/connexion/pull/530#issuecomment-344906519

The solution was add these parts on docs:
"Also check aiohttp handler examples"

So it is not a bug because the way how aiohttp handler works is different from the flask handler.
We can improve the documentation to expose this differences.

I can do an improve to support the same handler returns types as flask. But again, it is not a bug.

About the only_ony_api argument we can put in the docs improves too.

**Comment by prawn-cake** [comment]
@hjacobs 
I checked the changes with the [test project](https://github.com/prawn-cake/tryConnexion/blob/master/app.py) and also tried naive approach which didn't work. It worked without `only_one_api` though, didn't have problems with that.

@dutradda we should probably make a separate section with the few short examples, I feel that ppl will expect it to be transparent, so we need to disillusion it with clear examples / hello world aiohttp connexion app.

**Comment by dutradda** [comment]
@prawn-cake
I was thinking in improve the aiohttpApi.get_response to support others returns from the handlers.
These others returns will be the same as flask support, except a flaskResponse, of course.

Other approach is to implement these types of return types directly on the abstract class, so connexion will support natively.

**Comment by cognifloyd** [comment]
unifying response handling between Flask and AioHttp is implemented in #849.

**Comment by cognifloyd** [comment]
Even after #849 is merged, there will still be differences in serialization that will have to wait for v3.0 (WIP fix in #1095) because harmonizing serialization will require backwards incompatible changes for both flask and aiohttp.

**Comment by RobbeSneyders** [comment]
aiohttp has been dropped and switch between Flask and AsyncApp is now straightforward.
https://github.com/spec-first/connexion/issues/1395
